### PR TITLE
Cleanup rnc accession columns

### PR DESCRIPTION
**Pull request by blakesweeney**
This is the start of work to cleanup the columns in `rnc_accessions`. Generally, this pull request stops writing and loading specific columns which do not appear to be useful. This should be safe to just run, but as far as I can tell tests are broken right now. Prior to running this pipeline this we must manually edit `rnc_update.update_rnc_accession` function in the database to remove references to the columns cleaned up here. Otherwise it will break when loading things into the database. For reference the columns are:

- map
- allele
- ordinal
- pseudogene
- old_locus_tag
- anticodon
- division
- common_name
- classification
- species
- operon

The taxonomic columns are not needed but left in the entry object for now. There needs to be more careful work to remove them as we do use them for some logic within the pipeline.

### CODE REVIEW

File: files/import-data/load/accessions.ctl
Comment (afg1) [inline files/import-data/load/accessions.ctl:1]: Can we get rid of more? E.g. what are the `experiment` and `project` columns used for?

**Review by afg1 — APPROVED** [review]
I think this all looks ok. What percentage of the tests run now? We should probably try to get them working again to make this sort of thing easier in the future I guess

**Comment by blakesweeney** [comment]
I've been working through the database tests and have fixed some of them. Fixing those is going to take me a while. I can start tracking it but I suspect this is very slow going.

File: files/import-data/load/accessions.ctl
Comment (blakesweeney) [reply files/import-data/load/accessions.ctl:1]: I don't think experiment is really used, project might be somewhere. I will look into it and open another request to dump experiment too.

**Pull request merged**